great_expectations/expectations/expectation.py

```diff
@@ -2413,10 +2413,10 @@ def validate_configuration(
 
 
 class ColumnMapExpectation(TableExpectation, ABC):
-    map_metric = None
+    map_metric = ""
```

Comment:
```suggestion
    map_metric: str = ""
```

Should we say this must be a string?

Reply:
I ended up needing to revert this due to some `isinstance` checks on `str` vs `None` with error messages in this class.

Reply:
That sounds like incorrect logic.
An empty string shouldn't be a valid `map_metric`??